src/components/Tool/FixMailingAddresses/FixMailingAddresses.test.tsx

```diff
@@ -425,16 +438,7 @@ describe('FixMailingAddresses', () => {
 
       userEvent.click(secondaryAddresses[0]);
 
```

Comment:
Can you confirm that the primary has changed  ? I know there will be no query, but checking the UI has updated.